src/java.base/share/classes/java/time/Instant.java

```diff
@@ -850,14 +850,14 @@ public Instant plus(TemporalAmount amountToAdd) {
     public Instant plus(long amountToAdd, TemporalUnit unit) {
         if (unit instanceof ChronoUnit) {
             switch ((ChronoUnit) unit) {
```

Comment:
It looks like we need to put `return` here, otherwise `UnsupportedTemporalTypeException` will be thrown every time the method is called:
```
jshell> Instant.now()
$2 ==> 2021-06-22T15:35:28.771128667Z

jshell> $2.plus(10L, ChronoUnit.SECONDS)
|  Exception java.time.temporal.UnsupportedTemporalTypeException: Unsupported unit: Seconds
|        at Instant.plus (Instant.java:862)
|        at (#5:1)
```

Reply:
I've made that change now. See 5fa4e3c